src/content/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster.mdx

```diff
@@ -35,10 +35,10 @@ Here are some workload examples:
 
 Here's a workload:
 
-![Operational_workload.png](./images/Operational_workload.png "Operational_workload.png")
+![1_workload_health_tab.png](./images/1_workload_health_tab.png "workload_health_tab.png")
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com) > Apps > Workloads > (selected workload):** The workloads UI provides a curated view of how the entities in your workload are performing. The charts you see will depend on the types of entities you've included to the workload.
+  **[one.newrelic.com](https://one.newrelic.com) > Explorer > Workloads > (selected workload):** The workloads UI provides a curated view of how the entities in your workload are performing. The charts you see will depend on the types of entities you've included to the workload.
```

Comment:
"included in" or "added to" the workload?

Reply:
Done!